client/src/App.tsx

```diff
@@ -1,12 +1,22 @@
 import { Route, Routes } from 'react-router-dom';
 
+import { AddReviewPage } from './pages/AddReviewPage';
+import { CoursePage } from './pages/CoursePage';
+import { EditReviewPage } from './pages/EditReviewPage';
 import { Explore } from './pages/Explore';
 import { Home } from './pages/Home';
 
 const App = () => {
   return (
     <Routes>
-      <Route path='/' element={<Home />} />
+      <Route index element={<Home />} />
+      <Route path='course'>
+        <Route path=':id' element={<CoursePage />} />
+      </Route>
+      <Route path='review'>
+        <Route path=':id/add' element={<AddReviewPage />} />
+        <Route path=':id/edit' element={<EditReviewPage />} />
+      </Route>
```

Comment:
Is there a reason we chose to do pages instead of using dialogs? (https://headlessui.com/react/dialog)

Reply:
Dialogs add more complexity with state management, it also doesnt work super well on mobile (having to enter a lot of information in a dialog). It can be something we change later when improving UX.